### DISCUSSION TITLE
Can you release an updated version that fixes spammy `ViewportManager` logs?

### DISCUSSION THREAD

**Discussion by TheAfroOfDoom** [vscode-neovim/vscode-neovim#980] [Q&A] [answered]
Right now whenever I open a new file, I get the following error:
```
2022-07-29T00:16:15.842Z ViewportManager: No viewport for gridId: 1
```
This appears to have been ~fixed in https://github.com/vscode-neovim/vscode-neovim/commit/ff18dcda78e3bfe292e1866c4683373ce5fa2c7b.

It's quite distracting while coding and a new release to include it would be amazing.

---

Alternatively, I'm open to any solutions to suppress. My googling over the last few days hasn't led to much.

**Comment by zbw8388** [comment, marked as answer]
Sorry about the bug... One way to use the latest version would be getting it from CI. For example, you can download the patched extension in this [link](https://github.com/vscode-neovim/vscode-neovim/actions/runs/2754659505): 
![image](https://user-images.githubusercontent.com/60573138/181660946-3044ac9f-be8c-4af1-8e09-9531ae37f8db.png)

and then install it manually by clicking this button: 
![image](https://user-images.githubusercontent.com/60573138/181661167-90181886-9e2a-4c1f-8ee1-b8fe9c4f6537.png)

**Reply by TheAfroOfDoom** [reply]
Thanks for the quick response!
